uber/config.py

```diff
@@ -180,6 +180,8 @@ def __getattr__(self, name):
                 return True  # Defaults to unlimited stock for any stock not configured
             else:
                 return count_check < stock_setting
+        elif name.lower() in _config['secret']:
+            return _config['secret'][name.lower()]
```

Comment:
This is explained in a comment in ``configspec.ini`` but I'll reiterate here.  There are a bunch of options which we want to be able to access normally like ``c.SQLALCHEMY_URL`` and ``c.AWS_SECRET_KEY`` but which are extremely sensitive and we don't want them getting out.

How would they get out?  Well, there's a feature we make use of wherein we encode most of our config options into JSON and then send it to the browser.  (This was broken after the ``c`` refactor but I fixed it as part of this PR.)  This allows our Angular code to say things like ``c.ATTENDEE_BADGE`` the same as we do on the backend, which is great.  However, we certainly don't want to send sensitive info such as AWS creds.

The solution implemented here is to have a ``[secret]`` config section.  Anything under that section can be accessed same as before by saying ``c.AWS_SECRET_KEY`` or whatever, but it won't be rendered into the JSON we send to the browser.

I already moved all of our secret settings into that section, which you can see below.